coderedcms/templates/coderedcms/pages/base.html

```diff
@@ -196,5 +199,9 @@
             {% endblock %}
             {% block struct_seo_extra %}{% endblock %}
         {% endif %}
+
+        {% if settings.coderedcms.AnalyticsSettings.below_body_scripts %}
```

Comment:
Should probably wrap this in a block to make it easy to override if needed. E.g.

```
{% block body_tracking_scripts %}
```

Reply:
Done. Didn't know you could make up block names.